nbresuse/prometheus.py

```diff
@@ -18,14 +22,21 @@ def __init__(self, metricsloader: PSUtilMetricsLoader):
         self.metricsloader = metricsloader
         self.config = metricsloader.config
         self.session_manager = metricsloader.nbapp.session_manager
+        self.kernel_spec_manager = metricsloader.nbapp.kernel_spec_manager
 
-        gauge_names = ["total_memory", "max_memory", "total_cpu", "max_cpu"]
+        gauge_names = ["total_memory", "max_memory", "total_cpu", "max_cpu", "kernel_memory"]
         for name in gauge_names:
             phrase = name + "_usage"
-            gauge = Gauge(phrase, "counter for " + phrase.replace("_", " "), [])
+            if name is "kernel_memory":
```

Comment:
It looks like it would be better to use `==` here:
 
```suggestion
            if name == "kernel_memory":
```